tests/bitcoin/BitcoinClient.spec.ts

```diff
@@ -720,22 +735,23 @@ describe('BitcoinClient', async () => {
 
   describe('broadcastTransactionRpc', () => {
     it('should call the correct rpc with the input.', async (done) => {
-
+      const expectedIsWalletRpc = false;
```

Comment:
Same as the other comment, pull the definition scope to describe.
I think its safe to say this can be done on every "Rpc" wrapper function by now.